From: Qingqing <wangyiqing@kylinos.cn>
Date: Wed, 6 Aug 2025 07:12:54 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 100.0% (127 of 127 strings)

Translation: openkylin-nile-sp2/kom-service-support
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-sp2/kom-service-support/vi/
---
 .../translations/kom-service-support_vi.ts         | 144 ++++++++++-----------
 1 file changed, 72 insertions(+), 72 deletions(-)

diff --git a/plugins/service-support/translations/kom-service-support_vi.ts b/plugins/service-support/translations/kom-service-support_vi.ts
index 90ae930..6481fb0 100644
--- a/plugins/service-support/translations/kom-service-support_vi.ts
+++ b/plugins/service-support/translations/kom-service-support_vi.ts
@@ -6,12 +6,12 @@
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="35"/>
         <source>select detailed category</source>
-        <translation>vui lòng chọn danh mục chi tiết</translation>
+        <translation>Vui lòng chọn phân loại chi tiết</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="39"/>
         <source>System</source>
-        <translation>Sử dụng hệ thống</translation>
+        <translation>Trạng thái hệ thống</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="41"/>
@@ -26,12 +26,12 @@
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="41"/>
         <source>System crash</source>
-        <translation>Hệ thống sụp đổ</translation>
+        <translation>Hệ thống bị sập</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="42"/>
         <source>System performance</source>
-        <translation>Hiệu suất hệ thống</translation>
+        <translation>Hiệu năng hệ thống</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="42"/>
@@ -46,42 +46,42 @@
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="43"/>
         <source>System basis consulting</source>
-        <translation>Tư vấn cơ sở hệ thống</translation>
+        <translation>Tư vấn cơ bản về hệ thống</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="51"/>
         <source>Please describe in detail the problem you encountered, such as: unable to activate the system, can not find the relevant Settings, not clear system features, etc. </source>
-        <translation>Bạn có thể phản hồi tại đây về các vấn đề liên quan đến việc sử dụng hệ thống, ví dụ như: không thể kích hoạt hệ thống, không tìm thấy cài đặt liên quan, không rõ các tính năng tích hợp của hệ thống, v.v. Vui lòng mô tả chi tiết vấn đề bạn gặp phải, bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tệp để chúng tôi giải quyết vấn đề nhanh chóng . </translation>
+        <translation>Bạn có thể phản hồi các vấn đề về việc sử dụng hệ thống tại đây, ví dụ： Không thể kích hoạt hệ thống, Không thể tìm thấy cài đặt liên quan, Không rõ những chức năng mặc định của hệ thống v.v .Vui lòng mô tả chi tiết vấn đề bạn gặp phải, bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tài liệu liên quan để giúp chúng tôi giải quyết vấn đề một cánh nhanh. </translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="64"/>
         <source>Please describe in detail the problems you encountered, such as: peripheral connection failure, sharing function Settings, peripheral adaptation, etc. </source>
-        <translation>Bạn có thể phản hồi tại đây về các vấn đề liên quan đến thiết bị ngoại vi, ví dụ như: thiết bị ngoại vi không thể kết nối, cài đặt tính năng chia sẻ, tương thích thiết bị ngoại vi, v.v. Vui lòng mô tả chi tiết vấn đề bạn gặp phải, bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tệp để chúng tôi giải quyết vấn đề nhanh chóng . </translation>
+        <translation>Bạn có thể phản hồi các vấn đề về thiết bị ngoại vi tại đây, ví dụ: Thiết bị ngoại vi không thể kết nối Cài đặt chức năng chia sẻ ,tương thích thiết bị ngoại vi Vui lòng mô tả chi tiết nhất có thể sự cố gặp phải,bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tệp , giúp chúng tôi giải quyết vấn đề nhanh chóng. </translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="75"/>
         <source>Please describe in detail the problems that you encounter, such as obtaining, installing, and uninstalling Kirin software errors. </source>
-        <translation>Bạn có thể phản hồi tại đây về các vấn đề liên quan đến phần mềm ứng, ví dụ như:lỗi khi tải về, cài đặt và gỡ cài đặt phần mềm liên quan đến Kirin. Vui lòng mô tả chi tiết vấn đề bạn gặp phải, bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tệp để chúng tôi giải quyết vấn đề nhanh chóng . </translation>
+        <translation>Bạn có thể phản hồi các vấn đề liên quan đến phần mềm tại đây, chẳng hạn như: lỗi khi lấy, cài đặt và gỡ cài đặt phần mềm liên quan đến Kirin. Vui lòng mô tả chi tiết vấn đề bạn gặp phải, bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tệp để chúng tôi có thể nhanh chóng giải quyết vấn đề. </translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="85"/>
         <source>Please describe your problem in detail, or you can also fill in your request or comment here.</source>
-        <translation>Bạn có thể phản hồi về các vấn đề khác hoặc đưa ra ý kiến quý báu đối với chúng tôi ở đây.</translation>
+        <translation>Bạn có thể phản hồi các vấn đề khác hoặc đưa ra ý kiến quý báu cho chúng tôi tại đây.</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="57"/>
         <source>Peripheral</source>
-        <translation>Tư vấn thiết bị ngoại vi</translation>
+        <translation>Tư vấn về thiết bị ngoại vi</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="59"/>
         <source>Peripheral adaptation consulting</source>
-        <translation>Tư vấn thích ứng thiết bị ngoại vi</translation>
+        <translation>Tư vấn về tương thích thiết bị ngoại vi</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="59"/>
         <source>Peripheral driver acquisition</source>
-        <translation>Tải về trình điều khiển thiết bị ngoại vi</translation>
+        <translation>Tải xuống trình điều khiển thiết bị ngoại vi</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="60"/>
@@ -101,12 +101,12 @@
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="72"/>
         <source>Software use and error reporting</source>
-        <translation>Sử dụng và báo lỗi phần mềm</translation>
+        <translation>Sử dụng và báo lối phần mềm</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="81"/>
         <source>Other</source>
-        <translation>Khác</translation>
+        <translation>Rác khác</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="83"/>
@@ -124,7 +124,7 @@
     <message>
         <location filename="../service-support-backend/src/feedbackmanagerlogic.cpp" line="61"/>
         <source>Failed to create temporary directory!</source>
-        <translation>Tạo thư mực tạm thời thất bại!</translation>
+        <translation>Tạo thư mục tạm thời thất bại!</translation>
     </message>
 </context>
 <context>
@@ -132,7 +132,7 @@
     <message>
         <location filename="../UI/paginationwid.cpp" line="523"/>
         <source>Jump to</source>
-        <translation>Chuyển đến</translation>
+        <translation>Nhấp để chuyển đến</translation>
     </message>
     <message>
         <location filename="../UI/paginationwid.cpp" line="525"/>
@@ -155,7 +155,7 @@
     <message>
         <location filename="../UI/paginationwid.cpp" line="40"/>
         <source>Jump to</source>
-        <translation>Chuyển đến</translation>
+        <translation>Nhấp để chuyển đến</translation>
     </message>
     <message>
         <location filename="../UI/paginationwid.cpp" line="43"/>
@@ -173,17 +173,17 @@
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="192"/>
         <source>Machine</source>
-        <translation>Thông tin tổng thể máy</translation>
+        <translation>Thông tin máy tính</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="193"/>
         <source>Hardware</source>
-        <translation>Thông số phần cứng</translation>
+        <translation>Phần cứng</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="194"/>
         <source>Drive</source>
-        <translation>Thông tin Drive</translation>
+        <translation>Trình điều khiển card âm thanh</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="195"/>
@@ -198,17 +198,17 @@
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="197"/>
         <source>Network</source>
-        <translation>Trạng thái mạng lưới</translation>
+        <translation>Trạng thái mạng</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="198"/>
         <source>System</source>
-        <translation>Hệ thống</translation>
+        <translation>Trạng thái hệ thống</translation>
     </message>
     <message>
         <location filename="../plugin.cpp" line="18"/>
         <source>ServiceSupport</source>
-        <translation>Dịch vụ hỗ trợ</translation>
+        <translation>Hỗ trợ dịch vụ</translation>
     </message>
 </context>
 <context>
@@ -216,17 +216,17 @@
     <message>
         <location filename="../UI/uimainpage.cpp" line="82"/>
         <source>ServiceSupport</source>
-        <translation>Dịch vụ hỗ trợ</translation>
+        <translation>Hỗ trợ dịch vụ</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="85"/>
         <source>Multi-channel technical support services</source>
-        <translation>Dịch vụ hỗ trợ kỹ thuật đa kênh, giải quyết vấn đề và phản hồi ý kiến chỉ cần một cú nhấp chuột</translation>
+        <translation>Dịch vụ hỗ trợ kỹ thuật đa kênh ,giải quyết vấn đề và phản hồi ý kiến chỉ với một nhấp chuột</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="88"/>
         <source>Feedback</source>
-        <translation>Phản hồi vấn đề</translation>
+        <translation>Phản hồi</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="91"/>
@@ -251,7 +251,7 @@
     <message>
         <location filename="../UI/uimainpage.cpp" line="111"/>
         <source> to get more services</source>
-        <translation> ,tìm hiểu thêm về nội dung dịch vụ</translation>
+        <translation> Tìm hiểu thêm thông tin dịch vụ</translation>
     </message>
 </context>
 <context>
@@ -264,17 +264,17 @@
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="155"/>
         <source>Type</source>
-        <translation>Loại hình vấn đề</translation>
+        <translation>Loại card mạng</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="71"/>
         <source>bydesign</source>
-        <translation>Thiết kế là như vậy</translation>
+        <translation>Thiết kế là như thế</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="73"/>
         <source>duplicate</source>
-        <translation>vấn đề lặp lại</translation>
+        <translation>Vấn đề lặp lại</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="75"/>
@@ -294,17 +294,17 @@
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="81"/>
         <source>postponed</source>
-        <translation>Xử lý trì hoãn</translation>
+        <translation>Xử lý hoãn lại</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="83"/>
         <source>willnotfix</source>
-        <translation>từ chối giải quyết</translation>
+        <translation>Không giải quyết</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="99"/>
         <source>verify</source>
-        <translation>xác nhận giải quyết</translation>
+        <translation>Xác nhận giải quyết</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="102"/>
@@ -314,47 +314,47 @@
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="105"/>
         <source>Once identified, the issue will be closed and no further action will be taken.</source>
-        <translation>Sau khi xác nhận đã được giải quyết xong, vấn đề này sẽ được đóng lại và nhân viên hỗ trợ kỹ thuật sẽ không tiếp tục theo dõi nữa.</translation>
+        <translation>Sau khi xác nhận vấn đề đã được giải quyết, vấn đề sẽ được đóng lại và nhân viên kỹ thuật sẽ không theo dõi nữa.</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="107"/>
         <source>resolved</source>
-        <translation>giải quyết</translation>
+        <translation>Giải quyết</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="109"/>
         <source>cancel</source>
-        <translation>hủy</translation>
+        <translation>Hủy</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="155"/>
         <source>Description</source>
-        <translation>Mô tả vấn đề</translation>
+        <translation>Miêu tả</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="155"/>
         <source>Progress</source>
-        <translation>Tiến độ xử lý</translation>
+        <translation>Tiến độ</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="194"/>
         <source>No record</source>
-        <translation>Không có hồ sơ phản hồi</translation>
+        <translation>Không có ghi nhận phản hồi</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="195"/>
         <source>There is a network problem, please try again later</source>
-        <translation>Không thể kết nối với máy chủ, vui lòng thử lại sau</translation>
+        <translation>Kết nối đến máy chủ thất bại, Vui lòng thử lại sau</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="196"/>
         <source>Loading, please wait</source>
-        <translation>Đang tải ,vui lòng chờ đợi</translation>
+        <translation>Đang tải ,vui lòng chờ</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="209"/>
         <source>retry</source>
-        <translation>thử lại</translation>
+        <translation>Thử lại</translation>
     </message>
 </context>
 <context>
@@ -367,12 +367,12 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="114"/>
         <source>Type</source>
-        <translation>Loại hình vấn đề</translation>
+        <translation>Loại card mạng</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="132"/>
         <source>Please describe the problem in detail and you can upload a photo or file by clicking the button below.</source>
-        <translation>Vui lòng mô tả chi tiết vấn đề bạn gặp phải, bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tệp để chúng tôi giải quyết vấn đề nhanh chóng .</translation>
+        <translation>Vui lòng mô tả chi tiết vấn đề bạn gặp phải. Bạn cũng có thể nhấp vào nút bên dưới để tải lên hình ảnh hoặc tệp để giúp chúng tôi giải quyết vấn đề nhanh hơn.</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="134"/>
@@ -387,12 +387,12 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="172"/>
         <source>Details</source>
-        <translation>Mô tả vấn đề</translation>
+        <translation>Mô tả chi tiết</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="184"/>
         <source>ScreenShot</source>
-        <translation>Ảnh chụp màn hình</translation>
+        <translation>Chụp màn hình</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="200"/>
@@ -407,7 +407,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="260"/>
         <source>The phone number format is incorrect</source>
-        <translation>Định dạng số điện thoại không đúng</translation>
+        <translation>Lỗi định dạng số điện thoại</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="265"/>
@@ -417,12 +417,12 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="271"/>
         <source>appellation</source>
-        <translation>Danh xưng</translation>
+        <translation>Tên gọi</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="279"/>
         <source>Contact</source>
-        <translation>Thông tin liên lạc</translation>
+        <translation>Phương thức liên lạc</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="283"/>
@@ -432,7 +432,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="291"/>
         <source>The mailbox format is incorrect</source>
-        <translation>Định dạng email không đúng</translation>
+        <translation>Định dạng mailbox không hợp lệ</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="297"/>
@@ -448,7 +448,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="319"/>
         <source>Agree to take mine </source>
-        <translation>Đồng ý cho phép thu thập </translation>
+        <translation>Đồng ý lấy ... của tôi </translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="329"/>
@@ -468,12 +468,12 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="388"/>
         <source>Time period</source>
-        <translation>Khoảng thời gian</translation>
+        <translation>Thời gian xảy ra lỗi</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="423"/>
         <source>Information</source>
-        <translation>Thông tin hệ thống</translation>
+        <translation>Sao lưu và phục hồi</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="426"/>
@@ -483,7 +483,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="427"/>
         <source>days</source>
-        <translation>ngày</translation>
+        <translation>Ngày</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="439"/>
@@ -498,7 +498,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="471"/>
         <source>Upload log</source>
-        <translation>Nộp nhật ký</translation>
+        <translation>Đăng nhật ký</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="477"/>
@@ -508,7 +508,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="492"/>
         <source>Export to</source>
-        <translation>Xuất đến</translation>
+        <translation>Xuất sang</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="605"/>
@@ -518,7 +518,7 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="606"/>
         <source>Supported formats: </source>
-        <translation>Định dạng hỗ trợ: </translation>
+        <translation>Định dạng được hỗ trợ: </translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="133"/>
@@ -528,12 +528,12 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="237"/>
         <source>Files</source>
-        <translation>Tải lên tệp đính kèm</translation>
+        <translation>Tệp</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="686"/>
         <source>Repeat addition</source>
-        <translation>Thêm trùng lặp</translation>
+        <translation>Tên tệp đính kèm bị trùng lặp</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="702"/>
@@ -561,7 +561,7 @@
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="36"/>
         <source>OK</source>
-        <translation>Xác nhận</translation>
+        <translation>Chắc chắc</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="44"/>
@@ -571,23 +571,23 @@
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="92"/>
         <source>Submitted successfully</source>
-        <translation>Phản hồi của bạn đã được gửi thành công</translation>
+        <translation>Gửi thành công</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="96"/>
         <source>Cancel successfully</source>
-        <translation>Hủy bỏ thành công</translation>
+        <translation>Hủy thành công</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="100"/>
         <source>System is abnormal, contact technical support</source>
-        <translation>Đã phát hiện ra sự bất thường trong hệ thống của bạn, vui lòng liên hệ trực tiếp với nhân viên hỗ trợ kỹ thuật</translation>
+        <translation>Phát hiện hệ thống của bạn bị , vui lòng liên hệ trực tiếp với nhân viên kỹ thuật</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="101"/>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="108"/>
         <source>Log and submission is packed, please go</source>
-        <translation>Nhật ký và thông tin đã gửi đã được đóng gói. Vui lòng chuyển sang</translation>
+        <translation>Nhật ký và thông tin gửi đã được đóng gói, vui lòng truy cập vào địa chỉ sau</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="101"/>
@@ -598,17 +598,17 @@
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="106"/>
         <source>Submission failed</source>
-        <translation>Phản hồi của bạn gửi thất bại</translation>
+        <translation>Gửi thất bại</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="107"/>
         <source>Click &apos;Retry&apos; to upload again, or contact us directly.</source>
-        <translation>Nhấp vào &quot;Thử lại&quot; để tải lên lại, hoặc liên hệ trực tiếp với nhân viên hỗ trợ kỹ thuật để giải quyết.</translation>
+        <translation>Nhấp&quot;thử lại&quot; để tải lên lại hoặc liên hệ nhân viên kỹ thuật để giải quyết.</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="39"/>
         <source>Cancel</source>
-        <translation>Hủy bỏ</translation>
+        <translation>Hủy</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="129"/>
@@ -641,17 +641,17 @@
     <message>
         <location filename="../UI/uiselfservice.cpp" line="69"/>
         <source> to get more services</source>
-        <translation> , tìm hiểu thêm về nội dung dịch vụ</translation>
+        <translation> Tìm hiểu thêm thông tin dịch vụ</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="86"/>
         <source>Kylin technical services</source>
-        <translation>Tài khoản công chúng trên wechat</translation>
+        <translation>Dịch vụ kỹ thuật của phần mềm Kylin trên tài khoản công</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="61"/>
         <source>Jump to</source>
-        <translation>Nhấp để chuyến đến</translation>
+        <translation>Nhấp để chuyển đến</translation>
     </message>
 </context>
 <context>
@@ -659,12 +659,12 @@
     <message>
         <location filename="../UI/uiservicesupport.cpp" line="24"/>
         <source>Feedback</source>
-        <translation>Phản hồi vấn đề</translation>
+        <translation>Phản hồi</translation>
     </message>
     <message>
         <location filename="../UI/uiservicesupport.cpp" line="25"/>
         <source>Self service</source>
-        <translation>Dịch vụ tự phụ vụ</translation>
+        <translation>Dịch vụ tự phục vụ</translation>
     </message>
     <message>
         <location filename="../UI/uiservicesupport.cpp" line="27"/>
